From a852136fb9e4df9c60b8f3df96b94ea58958329a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 24 Jul 2020 12:45:17 +0200
Subject: [PATCH 1/1] dm: remove superfluous comment for union ofnode_union

"future live tree" does not make sense anymore as we have CONFIG_OF_LIVE.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/dm/ofnode.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dm/ofnode.h b/include/dm/ofnode.h
index a0d3df7786..8df2facf99 100644
--- a/include/dm/ofnode.h
+++ b/include/dm/ofnode.h
@@ -48,7 +48,7 @@ struct resource;
  *	is not a really a pointer to a node: it is an offset value. See above.
  */
 typedef union ofnode_union {
-	const struct device_node *np;	/* will be used for future live tree */
+	const struct device_node *np;
 	long of_offset;
 } ofnode;
 
-- 
2.27.0

